lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -73,7 +73,7 @@ def capture(money, authorization, options={})
         post = {}
         add_integration_key(post)
         post[:hash] = authorization
-        post[:amount] = amount(money)
+        post[:amount] = amount(money) if options[:include_capture_amount] == true
```

Comment:
We probably want `.to_s == 'true'` here. If this is sent as a string `"false"`, it's still going to trigger, but `to_s` is still true for a bool.

Reply:
That makes sense. Thanks for the review @curiousepic , the changes have been pushed.